### DISCUSSION TITLE
機器學習國際大會禁 ChatGPT！論文不得完全由 AI 模型生成

### DISCUSSION THREAD

**Discussion by ching020721** [1101-GS4719-C/Python-Community-GS4719-C#1031] [General]
這周機器學習國際大會正式宣布禁止使用ChatGPT產出「完全」一致的論文，可以參考或是編輯他產出的論文，但是該如何判定是經過編輯的，目前OpenAI正在研發中，不過ChatGPT並沒有參考資料來源，寫出來的論文若有錯誤的話，該論文就有點無法當作能正式參考的文獻了吧，還是希望大家是把ChatGPT當作幫助學習的工具，而不是投機取巧的作弊器，畢竟他厲害但不是自己的產出也沒有任何用處吧

https://www.inside.com.tw/article/30351-ICML-bans-ai-generated-papers

**Comment by manipulateed** [comment]
確實應當如此，論文應該是自己經過反覆實驗或是蒐集資料然後分析出來的，如果大家都以這種方式去寫論文，將會造成人沒有行動力與創造力。

**Comment by Jzd314** [comment]
論文如果用AI寫出來還算論文嗎?應該只能說是一篇AI寫出的心得文吧
